### PR TITLE
Update article_sap_trento.xml

### PR DESCRIPTION
**Pull request by abravosuse**
Adding a note saying tha the trento-supportconfig-plugin is available in the repos of SLES4SAP 15 SP3+ and can be installed from them using zypper.

### CODE REVIEW

**Review by tomschr — CHANGES_REQUESTED** [review]
Thanks @abravosuse, much appreciated!

I have some minor suggestions, hope you find them useful.

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -2477,6 +2477,7 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
           <step>
             <para>Download the Trento support plugin script:</para>
             <screen>&prompt.root;<command>wget</command> https://raw.githubusercontent.com/trento-project/helm-charts/main/scripts/trento-support.sh</screen>
+           <para>Note: customers of SLES4SAP 15 SP3+ can install the trento-supportconfig-plugin package directly from the official SLES4SAP 15 repos using zypper.</para>
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:2480]: Would it be possible to highlight the note in separate frame?

[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml:2480]: Yes, that's possible. If you think, this is a note that should be more visible you could write something like this:

```xml
<note>
  <!-- <title>Add a short summarizes about this note </title> -->
  <para>Note: customers of &sles4sap; 15 SP3 and higher can install the <package>trento-supportconfig-plugin</package> package directly from the official &sles4sapa; 15 repos using &zypper;.</para>
</note>
```

The title is optional. If you don't like or need it, just remove this line.

[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml:2480]: This is how it looks in HTML (from top to bottom: original, admonition, formalpara):

Screenshot_20240131_102649

[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:2480]: The highlighted Note looks great, @tomschr ! In that case, we obviously need to remove the paragraphs above and below with the note.

[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml:2480]: @abravosuse It was just an example that I did locally to show you the results. Makes comparison easier. :wink:  I didn't commit anything.

After reading the note, maybe we should provide a title. For example, "Package available with zypper" or something similar.

Would you like to amend the note or should I do the commit?

[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:2480]: Yes, we can add a title in the same fashion as the "Important" admonitions in the existing documentation. Something like:

`Note: package available in SUSE Linux Enterprise Server for SAP Applications 15 SP3 and higher`

and the the body of the note.

[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml]: @abravosuse I've added the `<note>` tag and the `<title>`. This is the result:

Screenshot_20240131_141628

**Review by abravosuse — COMMENTED** [review]
LGTM!

**Pull request merged**